frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -21,31 +23,165 @@ const AdminShoutouts: React.FC = () => {
+          actions={[
+            'Cancel',
+            {
+              key: 'hideShoutouts',
+              content: 'Hide Shoutout',
+              color: 'red',
+              onClick: () => onHide(shoutout)
+            }
+          ]}
+        />
+      );
+    return (
+      <Modal
+        trigger={<Button icon="eye slash" size="tiny" />}
+        header="Unhide Shoutout"
+        content="Are you sure that you want to show this shoutout?"
+        actions={[
+          'Cancel',
+          {
+            key: 'unhideShoutouts',
+            content: 'Unhide Shoutout',
+            color: 'red',
+            onClick: () => onHide(shoutout)
+          }
+        ]}
+      />
+    );
+  };
+
+  const DisplayList = (): JSX.Element => {
+    if (displayShoutouts.length === 0)
+      return (
+        <Card className={styles.noShoutoutsContainer}>
+          <Card.Content>No shoutouts in this date range.</Card.Content>
+        </Card>
+      );
+    if (view === 'PRESENT')
+      return (
+        <Item.Group divided>
+          {displayShoutouts.map((shoutout, i) => (
+            <Item key={i}>
+              <Item.Content>
+                <Item.Header
+                  className={styles.presentShoutoutTo}
+                >{`${shoutout.receiver}`}</Item.Header>
+                <Item.Meta
+                  className={styles.presentShoutoutFrom}
+                  content={` ${fromString(shoutout)}`}
+                />
+                <Item.Description
+                  className={styles.presentShoutoutMessage}
+                  content={shoutout.message}
+                />
+              </Item.Content>
+            </Item>
+          ))}
+        </Item.Group>
+      );
+    return (
+      <Item.Group divided>
+        {displayShoutouts.map((shoutout, i) => (
+          <Item key={i}>
+            <Item.Content>
+              <Item.Group widths="equal" className={styles.shoutoutDetails}>
+                <Item.Header className={styles.shoutoutTo}>{`${shoutout.receiver}`}</Item.Header>
+                <Item.Meta className={styles.shoutoutDate} content={dateString(shoutout)} />
+              </Item.Group>
+              <Item.Group widths="equal" className={styles.shoutoutHide}>
+                <Item.Meta className={styles.shoutoutFrom} content={fromString(shoutout)} />
+                <HideModal shoutout={shoutout} />
+              </Item.Group>
+              <Item.Description className={styles.shoutoutMessage} content={shoutout.message} />
+            </Item.Content>
+          </Item>
+        ))}
+      </Item.Group>
+    );
+  };
+
+  const ButtonPiece = (props: { shoutoutList: Shoutout[]; buttonText: string }): JSX.Element => {
+    const { shoutoutList, buttonText } = props;
+    return (
+      <Button
+        basic
+        color="grey"
+        onClick={() => {
+          setView(buttonText);
+          setDisplayShoutouts(shoutoutList);
+        }}
+        content={buttonText}
```

Comment:
I think there should be some indication (i.e. color change) of what option is currently selected.